setup.py

```diff
@@ -76,4 +76,6 @@ def read_version():
       packages=find_packages(),
       install_requires=install_requires,
       extras_require=extras_require,
-      include_package_data=True)
+      include_package_data=True,
+      # entry_points={'pytest11': ['aiomoto = tests.aws.aio.aiomoto_fixtures.py']},
```

Comment:
reminder need to resolve this

Reply:
added a commit to attempt to fix this using `aiobotocore.aiomoto`, but it might be an interim solution that could be substantially revised if some further thought into packaging might provide two packages from one repo.